test/UnitTests/ManageControllerTest.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Http.Core.Security;
+using Microsoft.AspNet.Http.Security;
+using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Fallback;
+using Xunit;
+using MusicStore.Models;
+
+namespace MusicStore.Controllers
+{
+    public class ManageControllerTest
+    {
+        private readonly IServiceProvider _serviceProvider;
+
+        public ManageControllerTest()
+        {
+            var services = new ServiceCollection();
+            services.AddEntityFramework()
+                    .AddInMemoryStore()
+                    .AddDbContext<MusicStoreContext>();
+
+            services.AddIdentity<ApplicationUser, IdentityRole>()
+                    .AddEntityFrameworkStores<MusicStoreContext>();
```

Comment:
yes.but..I think we could make something like the default constructor of DefaultHttpContext. The default constructor has been added later to support the unit test scenarios..